server/server.go

```diff
@@ -98,7 +109,7 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 		go func() {
 			logrus.Infof("Listening on %s:%d", opts.BindHost, httpPort)
 			err := httpServer.ListenAndServe()
-			if err != http.ErrServerClosed && err != nil {
+			if !errors.Is(err, http.ErrServerClosed) && err != nil {
```

Comment:
I'd prefer that you not change this. I know that the new way makes more sense (and were you writing this for the first time I would say this is a good change), but the fatal case causes the server to exit early. This could cause the server to exit in new cases (like if something was returning a wrapped `http.ErrServerClosed`).

Since this is strictly unnecessary for your PR, I'd rather leave this as-is, we can clean this up in another PR (like something that we don't intend on back-porting).

Reply:
ok, I was getting deprecation warnings from a linter I'm using, but sure...